diff --git usr/lib/plexmediaserver/Resources/Plug-ins-4e494e4/OpenSubtitles.bundle/Contents/Code/__init__.py usr/lib/plexmediaserver/Resources/Plug-ins-4e494e4/OpenSubtitles.bundle/Contents/Code/__init__.py
--- usr/lib/plexmediaserver/Resources/Plug-ins-4e494e4/OpenSubtitles.bundle/Contents/Code/__init__.py
+++ usr/lib/plexmediaserver/Resources/Plug-ins-4e494e4/OpenSubtitles.bundle/Contents/Code/__init__.py
@@ -210,6 +210,12 @@ def fetchSubtitles(proxy, token, part, imdbID=None, filename=None, season=None,
         if downloadQuota > 0:
 
           subData = Archive.GzipDecompress(subGz.content)
+          if l == "tur":
+            try:
+              subData = subData.decode("UTF-8")
+            except:
+              subData = subData.decode("ISO-8859-9")
+            subData = subData.encode("UTF-8")
           part.subtitles[Locale.Language.Match(st['SubLanguageID'])][subUrl] = Proxy.Media(subData, ext=st['SubFormat'])
           Log('Download quota: %d' % (downloadQuota))
 

