diff --git a/gtk/gui-thread-check/profiler/gui-thread-check.c b/gtk/gui-thread-check/profiler/gui-thread-check.c
--- a/gtk/gui-thread-check/profiler/gui-thread-check.c
--- b/gtk/gui-thread-check/profiler/gui-thread-check.c

@@ -93,6 +93,6 @@
 	
 	mono_profiler_install (NULL, NULL);
 	mono_profiler_install_enter_leave (simple_method_enter, NULL);
-	mono_profiler_set_events (MONO_PROFILE_ENTER_LEAVE);
+	// mono_profiler_set_events (MONO_PROFILE_ENTER_LEAVE);
 }
 
